decidim-templates/app/commands/decidim/templates/admin/update_proposal_answer_template.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Templates
+    module Admin
+      class UpdateProposalAnswerTemplate < Decidim::Command
+        # Initializes the command.
+        #
+        # template - The Template to update.
+        # form        - The form object containing the data to update.
+        # user        - The user that updates the template.
+        def initialize(template, form, user)
+          @template = template
+          @form = form
+          @user = user
+        end
+
+        def call
+          return broadcast(:invalid) unless @form.valid?
+          return broadcast(:invalid) unless @user.organization == @template.organization
+
+          @template = Decidim.traceability.update!(
+            @template,
+            @user,
+            name: @form.name,
+            description: @form.description,
+            field_values: { internal_state: @form.internal_state },
+            target: :proposal_answer
+          )
+
+          resource = identify_templateable_resource
+          @template.update!(templatable: resource)
+
+          broadcast(:ok, @template)
+        end
+
+        private
+
+        def identify_templateable_resource
```

Comment:
I see this is also duplicated in `decidim-templates/app/commands/decidim/templates/admin/create_proposal_answer_template.rb`. When we reach 3 we should find a way for DRYing this up (through a mixin, a concern or whatever). For the moment I can live with this